Update changelog.md (#4639)
mattermost/docs: source/administration/changelog.md

- **v5.34.4, release day TBD**
  - Investigating fixing an issue where missing query parameters in the datasource can cause MySQL servers to crash on startup. [MM-36236](https://mattermost.atlassian.net/browse/MM-36236)

>   - Fixing an issue where plugin icons are showing as a column instead of as a row in the left-hand side. [MM-36199](https://mattermost.atlassian.net/browse/MM-36199)

Reviewer: ```suggestion
  - Fixing an issue where plugin icons are displaying as a column instead of as a row on the left-hand side. [MM-36199](https://mattermost.atlassian.net/browse/MM-36199)
```
